src/components/flat-table/flat-table-row-header/flat-table-row-header.spec.tsx

```diff
@@ -312,15 +318,92 @@ describe("FlatTableRowHeader", () => {
               </tr>
             </thead>
           </table>
-        )
-          .find(StyledFlatTableRowHeader)
-          .getDOMNode();
+        );
+
+        const component = wrapper.find(StyledFlatTableRowHeader);
 
+        const element = component.getDOMNode();
         expect(element.getAttribute("data-sticky-align")).toEqual(
           stickyAlignment
         );
+
+        const composedPathMockFunction = jest.fn().mockImplementation(() => []);
+
+        const mockEvent = {
+          nativeEvent: {
+            composedPath: composedPathMockFunction,
+          },
+        };
+
+        component.simulate("focus", mockEvent);
+
+        expect(composedPathMockFunction).toHaveBeenCalled();
```

Comment:
**question**: given we are testing the z-index increase behaviour in the succeeding test, is asserting the internals of the focus event necessary here?

If this assertion is required for coverage reasons, moving this into a dedicated test would help with readability.

Reply:
This has been removed from the original test and a new test has been added with a clear description in order to improve readability.